### DISCUSSION TITLE
Installing ng-zorro component individually

### DISCUSSION THREAD

**Discussion by amitsinghrawat1994** [NG-ZORRO/ng-zorro-antd#6528] [Q&A] [answered]
I am new in NG-ZORRO.
My question is can we install a particular ng-zorro component individually or I have to install fully ng-zorro-antd ?

I have used ngx-bootstrap in past and we can also install a single individual component and use them.

**Comment by wzhudev** [comment, marked as answer]
We publish ng-zorro-antd as a whole and don't support install individual component. But you could always use secondary entry points to import components. I suppose that it would feed seemlessly? And thanks to tree-shake, it wouldn't boom your package size.

**Comment by amitsinghrawat1994** [comment]
Thanks, @wendellhu95. let me check how I can include ng-zorro-antd in my project.

**Reply by amsaighi** [reply]
> Thanks, @wendellhu95. let me check how I can include ng-zorro-antd in my project.

Try to follow this steps [Get started](https://ng.ant.design/docs/getting-started/en)
npm install ng-zorro-antd
or 
yarn install ng-zorro-antd
don't forget the add style ng-zorro [Import styles to your project ](https://ng.ant.design/docs/getting-started/en#import-styles)
